### PR TITLE
[P&P] Display storage warning when repo storage almost full or exceeded

### PR DESCRIPTION
**Pull request by pjlast**
When an instance is nearing its storage limit for repositories, or if the storage limit has been exceeded, then a warning banner will appear for site admins, and a warning will be displayed on the Repositories site admin page.

Storage full:
image

Storage almost full:
image

## Test plan

Manual tests (shown in screenshots).

### CODE REVIEW

File: client/web/src/site-admin/SiteAdminRepositoriesPage.tsx [hunk 1]
```diff
@@ -224,6 +238,20 @@ export const SiteAdminRepositoriesPage: React.FunctionComponent<React.PropsWithC
                 </Link>
                 .
             </Text>
+            {licenseInfo != null && (licenseInfo.codeScaleCloseToLimit || licenseInfo.codeScaleExceededLimit) && (
+                <Alert variant={licenseInfo.codeScaleExceededLimit ? 'danger' : 'warning'}>
+                    <H4>
+                        {licenseInfo.codeScaleExceededLimit ? (
+                            <>You've used all 100GB of storage</>
```
Comment (unknwon) [inline client/web/src/site-admin/SiteAdminRepositoriesPage.tsx:245]:
```suggestion
                            <>You've used all 100GiB of storage</>
```

Just to be consistent with the site alerts?

File: client/web/src/site-admin/SiteAdminRepositoriesPage.tsx [hunk 2]
```diff
@@ -224,6 +238,20 @@ export const SiteAdminRepositoriesPage: React.FunctionComponent<React.PropsWithC
...
+                        ) : (
+                            <>Your Sourcegraph is almost full</>
+                        )}
+                    </H4>
+                    {licenseInfo.codeScaleExceededLimit ? <>You're about to reach the 100GB storage limit. </> : <></>}
```
Comment (unknwon) [inline client/web/src/site-admin/SiteAdminRepositoriesPage.tsx:250]:
```suggestion
                    {licenseInfo.codeScaleExceededLimit ? <>You're about to reach the 100GiB storage limit. </> : <></>}
```

**Review by unknwon — APPROVED** [review]
🚀

File: client/web/src/site-admin/SiteAdminRepositoriesPage.tsx (hunk 1 above)
Comment (pjlast) [reply client/web/src/site-admin/SiteAdminRepositoriesPage.tsx:245]: Yeah makes sense, good spot!

**Pull request merged**